commands/FBTextInputCommands.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+
+# Copyright (c) 2016, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+import os
+
+import lldb
+import fblldbbase as fb
+import fblldbviewhelpers as viewHelpers
+
+ACCESSIBILITY_ID = 0
+REPLACEMENT_TEXT = 1
+INPUT_TEXT = 0
+
+
+def lldbcommands():
+  return [
+    FBInputTexByAccessibilityIdCommand(),
+    FBInputTexToFirstResponderCommand(),
+  ]
+
+
+class FBInputTexByAccessibilityIdCommand(fb.FBCommand):
+  def name(self):
+    return 'setxt'
+
+  def description(self):
+    return 'Set text on text on a view by accessibility id.'
+
+  def args(self):
+    return [
+      fb.FBCommandArgument(arg='accessibilityId', type='string', help='The accessibility ID of the input view.'),
+      fb.FBCommandArgument(arg='replacementText', type='string', help='The text to set.')
+    ]
+
+  def run(self, arguments, options):
+    self.findView(rootView(), arguments[ACCESSIBILITY_ID], arguments[REPLACEMENT_TEXT])
+
+  def findView(self, view, searchIdentifier, replacementText):
+    views = subviewsOfView(view)
+    for index in range(0, viewsCount(views)):
+        subview = subviewAtIndex(views, index)
+        self.findView(subview, searchIdentifier, replacementText)
+    else:
+      identifier = accessibilityIdentifier(view)
+      if isEqualToString(identifier, searchIdentifier):
+        setTextInView(view, replacementText)
+
+
+class FBInputTexToFirstResponderCommand(fb.FBCommand):
+  def name(self):
+    return 'input'
```

Comment:
I chose input because this acts like someone inputing text into a text field/text view which has the cursor in it. 

I could change this to `inputtext` and the previous command to `settext`. 

I went with terse commands based on whats currently there in lldb and chisel. Again, no hard opinion on what these should be called.